src/content/developers/docs/transactions/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Transactions
 description: An overview of Ethereum transactions – how they work, their data structure, and how to send them via an application.
-lang: en
+lang: en-UK
```

Comment:
Our internationalization setup does not yet account for UK English, just 'en'. Mind reverting this?